src/types/tcb_info.rs

```diff
@@ -350,40 +362,50 @@ impl Tcb {
                 v2.sgxtcbcomp15svn,
                 v2.sgxtcbcomp16svn,
             ],
-            Self::V3(v3) => v3.sgxtcbcomponents.map(|comp| comp.svn),
+            Self::V3(v3) => {
+                let mut result = [0u8; 16];
+                for i in 0..16 {
+                    result[i] = v3.sgxtcbcomponents[i].svn;
```

Comment:
I don't think this helps. By default all primitive types are copied, removing the copy on `TcbComponentV3` and doing this is exactly same.

Reply:
`Copy` trait was removed because otherwise I would have to implement the trait for `Option<String>`